server/ruby/config_helper.rb

```diff
@@ -0,0 +1,264 @@
+# This utility is to help ensure the sample is configured correctly when first
+# getting started.
+#
+# Refer to `server.rb` for integration details.
+require 'stripe'
+
+class ConfigHelper
+  REQUIRED_VARS = [
+    'BASIC_PRICE_ID',
+    'DOMAIN',
+    'PRO_PRICE_ID',
+    'STATIC_DIR',
+    'STRIPE_PUBLISHABLE_KEY',
+    'STRIPE_SECRET_KEY',
+    'STRIPE_WEBHOOK_SECRET',
+  ]
+
+  def self.check_env!(*args)
+    helper = self.new(*args)
+
+    # Confirms the required environment variables have
+    # been configured.
+    valid = helper.dotenv_exists?
+
+    # Confirms API keys are set and have the expected
+    # prefixes.
+    valid &= helper.valid_api_keys?
+
+    exit if !valid
+
+    # Once we've done basic key validation, we can set the API
+    # key and make deeper assumptions.
+    Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+
+    exit if !helper.valid_paths?
+    exit if !helper.valid_domain?
+
+    exit if !helper.valid_prices?
+  end
+
+  def initialize(*args)
+    @vars = Dotenv.parse(*args)
+  end
+
+  def write_dotenv!
+    File.open('.env', 'w') do |f|
+      REQUIRED_VARS.each do |v|
+        f.puts "#{v}=#{@vars[v]}"
+      end
+    end
+  end
+
+  def valid_prices?
+    _valid_price?('BASIC_PRICE_ID', 1200) &&
+      _valid_price?('PRO_PRICE_ID', 1800)
+  end
+
+  def _valid_price?(price_id, unit_amount)
+    price = ENV[price_id]
+    if price.nil? || price == '' || price.include?('...')
+      puts <<~DOC
+        Environment variable missing for #{price_id}. This should be the ID
+        of a Price object in your Stripe account. You can create a product and related prices in the dashbaord here: https://dashboard.stripe.com/test/products.
+
+        Would you like us to create a Price now? [Y/n]
+      DOC
+      c = gets.chomp
+      if c.upcase == "N"
+        puts "Please create a Product and Price from the dashboard, then set its ID in .env."
```

Comment:
Nit, but ".env." is tricky to read/parse. A slight rephrasing to "...in your .env file." removes the punctuation conflict.